src/dash-table/derived/style/py2jsCssProperties.ts

```diff
@@ -557,6 +559,7 @@ export default new Map<string, string>([
     ['borderLeftStyle', 'borderLeftStyle'],
     ['borderLeftWidth', 'borderLeftWidth'],
     ['borderRight', 'borderRight'],
+    ['borderRadius', 'borderRadius'],
```

Comment:
Adding support for `border-radius` and the Python and React flavors

Reply:
whew, exciting file... any reason this couldn't be rewritten DRYer? Like:
```js
const attrParts = [
    // each attr only once
    ['align', 'content'],
    ['align', 'items'],
    ['alignment', 'adjust'],
    ...
];

const makeMap = attrParts => {
   // generate the tripled list in all formats
}

export default new Map(makeMap(attrParts));
```
Wouldn't need to be done now, but seems more maintainable and efficient.

Reply:
Actually this file is generated - forgot to push the generator update.

Goal of the generator is to prevent having to eval all of this at runtime and only have a table mapping for python/css/react casing. That said, your comment below (https://github.com/plotly/dash-table/pull/568#discussion_r320931811) is correct. Should generate a resolve array of these values instead of doing it on the fly/